README.md

```diff
@@ -1 +1,65 @@
 # org
+
+This repository contains company-wide guidelines on writing code and organizing
+workflow. If Kowainik member don't know how some part of his/her work should be
+he/she should refer to this guide. If some part of workflow is not clear from
+this document — open issue to this repository. But, of course, everyone should
```

Comment:
let's make it a bit with *feel free to open an issue* or *please open an issue*, what do you think?

Reply:
I agree. This will make people more comfortable with creating new issues. I'm for first option.